src/promptflow/tests/executor/e2etests/test_traces.py

```diff
@@ -359,7 +375,12 @@ def test_otel_trace_with_prompt(
         prompt_tpl_file,
     ):
         execute_function_in_subprocess(
```

Comment:
I guess we can directly change the implementation of `execute_function_in_subprocess`?